### PR TITLE
Governance Responsiveness

### PR DESCRIPTION
**Pull request by arjunu**
To be merged after https://github.com/Neufund/platform-frontend/pull/4470

### CODE REVIEW

**Comment by arjunu** [comment]
> while the General Information page fits the screen well, the Overview doesn't fit the screen and the user needs to scroll horizontally.
> 
> reproducible with Metamask app (iPhone XS) and dev tools (any device chosen).
> fixture used - Issuer_payouts
> 
> image

Fixed now. It was supposed to be two columns in small screens.

File: packages/web/app/components/governance/Governance/GeneralInformation/GeneralInformation.module.scss [hunk 1]
```diff
@@ -1,21 +1,39 @@
 .title-row {
   display: flex;
   justify-content: space-between;
+  margin-bottom: px-to-rem(20x);
```
Comment (AlexeyTuboltsev) [inline packages/web/app/components/governance/Governance/GeneralInformation/GeneralInformation.module.scss:4]: 20x looks like a typo?

File: packages/web/app/components/appRoutes.ts [hunk 2]
```diff
@@ -102,3 +104,5 @@ export const appRoutes = {
    */
   etoPublicViewLegacyRoute: "/eto/view/:previewCode",
 } as const;
+
+export type TAppRoute = Values<typeof appRoutes>;
```
Comment (AlexeyTuboltsev) [inline packages/web/app/components/appRoutes.ts:108]: 👍

File: packages/web/app/components/governance/Governance/GovernancePageSelect.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+type TOptionType = { label: string; value: string };
+
+type TProps = {
+  value: TAppRoute;
+  onChange: (to: string) => void;
+};
+
+export const GovernancePageSelect: React.FunctionComponent<TProps> = props => (
+  <VirtualizedSelect
```
Comment (AlexeyTuboltsev) [inline packages/web/app/components/governance/Governance/GovernancePageSelect.tsx:17]: what are the benefits of virtualizedSelect here? Why not a simple select? There are only 6 options, not 6000 where virtualizedSelect makes sense
Or maybe I miss something

File: packages/web/app/components/governance/Governance/GovernancePageSelect.tsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+      page =>
+        ({
+          label: page.label,
+          value: page.to as string,
+        } as TOptionType),
+    )}
+    value={props.value}
+    searchable={false}
+    clearable={false}
+    onChange={(selectedOption: unknown) => {
```
Comment (AlexeyTuboltsev) [inline packages/web/app/components/governance/Governance/GovernancePageSelect.tsx:29]: is selectedOption really unknown here? 
there are many type castings in this file, but all arguments are actually pretty well known. Are castings really necessary?

Comment (AlexeyTuboltsev) [reply packages/web/app/components/governance/Governance/GovernancePageSelect.tsx:29]: btw as TSC point out, selectedOption may be null. Need to add a check for this.

Comment (AlexeyTuboltsev) [reply packages/web/app/components/governance/Governance/GovernancePageSelect.tsx:29]: hm, the typings of virtualizedSelect don't seem to be correct at all. I strongly suggest to get rid of it and use a simple html select. A check for null would be necessary there as well

File: packages/web/app/components/governance/Governance/GeneralInformation/GeneralInformation.module.scss (hunk 1 above)
Comment (arjunu) [reply packages/web/app/components/governance/Governance/GeneralInformation/GeneralInformation.module.scss:4]: nice catch, fixed

File: packages/web/app/components/governance/Governance/GovernancePageSelect.tsx (hunk 4 above)
Comment (arjunu) [reply packages/web/app/components/governance/Governance/GovernancePageSelect.tsx]: as per discussion, put it in an unsafe file for now with the type castings. And here is the issue to create a new component: https://github.com/Neufund/platform-frontend/issues/4535

**Pull request merged**